### PR TITLE
new icon: tableau (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #1030: Icon request: Tableau
### About the icon

Tableau is a data visualisation tool used to create visual dashboards for business reports and business intelligence

### Links and sources

Website link: https://www.tableau.com/

Official resource for the icon: https://www.tableau.com/about/media-download-center
https://cdnl.tblsft.com/sites/default/files/pages/tableau_cmyk_2015.png

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Gautam25Raj**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1030**

## Link to prove your SVG is correct and up-to-date.

https://www.tableau.com/about/media-download-center
https://cdnl.tblsft.com/sites/default/files/pages/tableau_cmyk_2015.png

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -9300,6 +9300,29 @@
             }
         ]
     },
+    {
+        "name": "tableau",
+        "altnames": [],
+        "tags": [
+            "data visualization",
+            "software",
+            "business intelligence"
```
Comment (Panquesito7) [inline devicon.json:9309]: Looks like the project is open-source as well: https://github.com/tableau.
```suggestion
            "business intelligence",
            "open-source"
```

**Review by Gautam25Raj — COMMENTED** [review]
Updated devicon.json

**Review by Panquesito7 — APPROVED** [review]
Awesome work. Thank you! 🚀

image

image

**Review by lunatic-fox — APPROVED** [review]
Great work! ✔️
Thank you for this contribution! 🚀

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Should we allow this icon @Panquesito7?
I think this is a case of a CRM tool: [Wiki](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F)

**Comment by Panquesito7** [comment]
You're right. The Wiki specifically states "Salesforces", and Tableu appears to be a product from them.
I'll be closing this PR for now. We can still discuss further about the icon addition.

Thank you for your contributions. 🙂

**Pull request closed without merging**